src/main/java/org/fundacionjala/at15/katas/pokerhands/mauricio/Game.java

```diff
@@ -0,0 +1,75 @@
+package org.fundacionjala.at15.katas.pokerhands.mauricio;
+
+public class Game {
+    private PokerHand h1;
+    private PokerHand h2;
+    private final int tie3 = 3;
+    private final int tie4 = 4;
+
+    public Game(PokerHand hand1, PokerHand hand2) {
+        this.h1 = hand1;
+        this.h2 = hand2;
+    }
+
+    public String whoWins() {
+        String winner = "";
+
+        if (flushCards() == 1) { //Ordered from high to low poker hand rank
```

Comment:
Even if checkstyle is not complaining, it seems that 1 and 2 are working as magic numbers here because they do not describe what they mean. It would be good to define constants that describe their meaning (example, PLAYER_1_HAND/PLAYER_2_HAND)